src/components/CertificateSettings.vue

```diff
@@ -0,0 +1,195 @@
+<!--
+  - @copyright 2023 Richard Steinmetz <richard@steinmetz.cloud>
+  -
+  - @author 2023 Richard Steinmetz <richard@steinmetz.cloud>
+  - @author 2023 Hamza Mahjoubi <hamzamahjoubi221@gmail.com>
+  -
+  - @license AGPL-3.0-or-later
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<template>
+	<div>
+		<Multiselect
+			:allow-empty="false"
+			:options="aliases"
+			:searchable="false"
+			:value="alias"
+			label="name"
```

Comment:
```suggestion
			:value="alias"
			:placeholder="t('mail', 'Select an alias')"
			label="name"
```

Otherwise, the placeholder will be `Select option` which is a bit too generic.